Lombiq.Hosting.Tenants.FeaturesGuard/Models/ConditionallyEnabledFeaturesOptions.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+
+namespace Lombiq.Hosting.Tenants.FeaturesGuard.Models;
+
+public class ConditionallyEnabledFeaturesOptions
+{
+    // Needs to be settable for binding.
+#pragma warning disable CA2227 // Collection properties should be read only
+    public IDictionary<string, string> EnableFeatureIfOtherFeatureIsEnabled { get; set; }
```

Comment:
The value type would be more applicable with a string array or `IEnumerable<string>` than comma-separated string values.